modules/imgproc/test/test_color.cpp

```diff
@@ -3072,20 +3072,40 @@ TEST(ImgProc_RGB2YUV, regression_13668)
     EXPECT_EQ(res, ref);
 }
 
-TEST(ImgProc_cvtColorTwoPlane, missing_check_17036)  // test can be removed if required feature is implemented
+TEST(ImgProc_cvtColorTwoPlane, regression_17036)
 {
-    std::vector<uchar> y_data(700 * 480);
-    std::vector<uchar> uv_data(640 * 240);
+    RNG rng;
```

Comment:
It's better to use test system controlled random generator:
```
RNG& rng = ts->get_rng();
```

Reply:
Looks like ::testing::Test parent class has no pointer to the system.

Reply:
`cv::theRNG()` should be used (controlled by OpenCV test system).